style2.css

```diff
@@ -0,0 +1,49 @@
+.burger {
+	background-color: salmon;
+	display: none;
+	height: 3px;
+	position: relative;
+	width: 10px;
+}
+
+.burger:before {
+	content: " ";
+	background-color: salmon;
+	height: 3px;
+	position: absolute;
+	top: 5px;
+	width: 10px;
+}
+
+.burger:after {
+	bottom: 5px;
+	content: " ";
+	background-color: salmon;
+	height: 3px;
+	position: absolute;
+	width: 10px;
+}
+
+@media (min-width: 319px) {
+	.nav {
+		display: none;
+	}
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/CSS17K/pullRequest?prid=5813015)